docs/UserGuide.md

```diff
@@ -293,9 +293,9 @@ Format:
 
 ## FAQ
 
-**Q**: How do I transfer my data to another computer? 
+**Q**: Does History Flashcard have GUI version?
 
-**A**: Well, write the User Guide in active voice anyway.
+**A**: Not at this moment. But we are working on it - keep a lookout for our new releases!
```

Comment:
Haha, but I don't think a GUI will ever happen??

Reply:
I cannot think of other things to add in FAQ lol So I just add some random stuff. New release can be v3.0 which we'll never work on hahahahah

Reply:
Hmm one possible FAQ:
Where can I find the files containing all my flashcard information?
In folder xxx, the file xxx.txt stores the flashcards in the following format: ...
It's info that doesn't really fall under any of the commands, but some users may want to know.

Reply:
> Hmm one possible FAQ:
> Where can I find the files containing all my flashcard information?
> In folder xxx, the file xxx.txt stores the flashcards in the following format: ...
> It's info that doesn't really fall under any of the commands, but some users may want to know.

Sounds good, I'll update :)